install/lsstsw.rst

```diff
@@ -50,7 +50,7 @@ The ``deploy`` script automates several things for you:
 4. clones versiondb_, a robot-made Git repository of package dependency information, and
 5. creates an empty Stack *installation* directory, :file:`stack/`.
 
-``lsstsw`` `clones repositories using HTTPS <https://github.com/lsst/lsstsw/blob/master/etc/repos.yaml>`_.
+``lsstsw`` `clones repositories using HTTPS <https://raw.githubusercontent.com/lsst/repos/master/etc/repos.yaml>`_.
```

Comment:
Could you change this to read

```rst
``lsstsw`` clones repositories using HTTPS (`see repos.yaml <https://github.com/lsst/repos/blob/master/etc/repos.yaml>`_.
```

Looking now, think that sentence will work better. And using the GitHub `blob` link is nice because it gives the reader context about where the file lives. `raw.githubusercontent.com` is good for letting someone download a file quickly.